### PR TITLE
Add guides for MongoDB connector

### PR DESCRIPTION
**Pull request by tomasz-sadura**
Add guides for the official MongoBD sink and source connector.

Ref.: https://www.mongodb.com/docs/kafka-connector/current/

### CODE REVIEW

File: backend/pkg/connector/interceptor/mongo_hook.go [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+			if hasKafkaConnectConfigProvider(config["connection.password"].(string)) {
+				config["connection.uri"] = strings.ReplaceAll(config["connection.uri"].(string), passwordPlaceholder, config["connection.password"].(string))
+			}
+		}
+	}
+}
+
+func setFormatOutputStream(config map[string]any) {
+	for _, field := range []string{"key", "value"} {
+		if config["output.format."+field] == nil {
```
Comment (bochenekmartin) [inline backend/pkg/connector/interceptor/mongo_hook.go:60]: Imho, whitespaces are missing between + and a string

**Review by bochenekmartin — APPROVED** [review]

**Review by bochenekmartin — APPROVED** [review]
Consider squashing the commits

File: backend/pkg/connector/interceptor/mongo_hook.go (hunk 1 above)
Comment (tomasz-sadura) [reply backend/pkg/connector/interceptor/mongo_hook.go:63]: linter does not allow it
```pkg/connector/interceptor/mongo_hook.go:63: File is not `gci`-ed with --skip-generated -s standard,default,prefix(github.com/redpanda-data/console/backend) (gci)
		if config["output.format." + field] == nil {
task: Failed to run task "backend:lint": exit status 1```

Comment (bochenekmartin) [reply backend/pkg/connector/interceptor/mongo_hook.go:63]: okay, disregard then 🙂

**Pull request merged**